server/server.go

```diff
@@ -823,7 +825,11 @@ func (s *Server) processTrustedKeys() bool {
 				return false
 			}
 		}
-		s.trustedKeys = s.opts.TrustedKeys
+		tk := make([]string, len(s.opts.TrustedKeys))
```

Comment:
If this is a copy you want to make, I believe a more idiomatic way would be:
```
s.trustedKeys = append([]string(nil), s.opts.TrustedKeys...)
```
Not sure if Go optimize and pre-allocate the right size (which it probably does), but regardless this is not a fast path (executed only once), so the more compact form is better I think.

Reply:
that's neat, fixed